### DISCUSSION TITLE
[2주차 알고리즘 스터디 회고] 구현

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/js-algorithm-basecamp#31] [General]
## 알고리즘 스터디 회고
주차: 2주차
주제: 그리디
일시: 2021-10-10 Sun 22:00 ~ 23:00
## 회고 순서
1. 체크인

2. 리뷰: ~22:35
- 특별히 조원 정하지 않고 자유롭게 서로 PR 리뷰해주세요
- 학습을 아예 안하신분들도 리뷰하셔도 좋고 아니면 개인학습 하셔도 좋아요 :+1:

**Comment by bigsaigon333** [comment]
체크인
오늘의 기분을 10점 만점으로 작성해주세요

**Reply by bigsaigon333** [reply]
**동동**  6점

방금 다른 스터디가 끝나서 정신이 없네요... 이번주 많이 공부를 못해서 좀 마음이 찝찝합니다 ㅠㅠ 
회고때 자기반성 철저히 할 생각이에요 ㅎㅎ

**Reply by Sunmon** [reply]
Sunmon 7점
내일 하루 더 쉬어서 좋습니다

**Reply by skt-t1-byungi** [reply]
8 낼쉬어서 기분좋음. 오늘 뭔가 생산적인 걸 한게 없어서 찝찝

**Reply by Yeonlisa** [reply]
8점 : 역시 알고리즘은 힘들어요

**Reply by ganeodolu** [reply]
호준 6점 : 자가격리 중이라 활동범위가 더 좁아져서 방콕 중 이네요 :scream:

**Reply by LeeGeonwoo22** [reply]
5.5 점 

안녕하세요 알고리즘 스터디클럽 들어오구 첫 회고입니다.
알고리즘의 청정수라 문제도 계속 읽기만하고 , 
공부법 찾아보고, 개념공부하고 그렇게 보낸것 같네요..ㅠ 
저는 5.5점으로 하겠습니다..!!! 
일단은 운동을 하고와서인지 찌뿌듯하고 졸리네요 ㅎㄷㄷ..

**Reply by sooster910** [reply]
Sue 3점 
밸런스잡기가 쉽지가 않네요 ㅠ  밀려서 부랴부랴 하게 되네요.

**Reply by hyunhaha** [reply]
7점 계획한 만큼 다 못해서..😓

**Comment by bigsaigon333** [comment]
## 회고 작성: ~22:50
- 이번 주차 스터디하면서 좋았던 점 작성
- 이번 주차 스터디하면서 어려웠던 점 작성
- 오늘 회고 때 가장 인사이트가 있었던 점 
- 스터디 건의사항
- action plan: 회고를 통해 얻은 다음 주차 스터디 때 적용하고 싶은 행동 방안 작성

**Reply by bigsaigon333** [reply]
## 동동

### 이번 주차 스터디하면서 좋았던 점 작성
스터디 준비하면서 버퍼로 몇개 챕터를 미리 풀어놨어서 아예 빼먹지는 않을 수 있었네요..
그리고 그루밍님이랑 목요일 저녁에 광장에서 모각알 했었는데, 시간 정해놓고 하자고 하니까, 강제로라도 하게 되어서 좋았습니다 :+1: 

### 이번 주차 스터디하면서 어려웠던 점 작성
- 팀 프로젝트, 모의면접 스터디 등등등... 이번주 왤케 바빴는지 모르겠네요. 막상 뭐했는지 생각해보니까 떠오르는 건 없는데..
- 언제 뭘 했는지 한번 되뇌이는 주간 회고 시간이 필요합니다 ~0 ~..
- 네이버 공채 서류탈락하고 착잡했지만... 다시 멘탈 챙겼습니다 ㅎㅎ.. 
- 빗썸코리아 코테 3개중에 2개는 푼거 같은데, 결과가 안나와서.... 어케될런지 궁금하네요 ㅠㅠ

### 오늘 회고 때 가장 인사이트가 있었던 점
호준님이 그리디랑 구현이랑 모두 다 푸셔서 인상적이었습니다. 실전문제도 많은 분들이 푸셨고, leetcode도 푸신 sunmon 님.. 
너무 대단합니다..! 
자극 됩니다!!

### 스터디 건의사항
- 회고 공지 미리 올리겠습니다 ㅠ 전날에는 올렸어야 했는데.. 허
- 연리사님이 의견 주셨던 방안도 스터디에 적용할 수 있도록 어나운스하겠습니다~!

### action plan: 회고를 통해 얻은 다음 주차 스터디 때 적용하고 싶은 행동 방안 작성
- 언제 알고리즘 공부할지를 미리 좀 정해둘 예정입니다. 그냥 시간 남으면 해야지 했다가... 시간이 안남네요. 
  공부할 시간을 미리 정해두자..!
- 실전 문제 다 풀진 않더라도 하나 이상 풀자...

**Reply by ganeodolu** [reply]
호준 
- 좋은 점 : 다른 사람의 코드를 보고, 리뷰를 받으며 새로 깨달을 수 있는 점
- 어려운점 : 문제가 이해가 안되는 것들이 있어서 시간내 코드를 많이 보지 못한 점
- action plan :  찜찜했던 것들 다시 보기

**Reply by Yeonlisa** [reply]
### 이번 주차 스터디하면서 좋았던 점 작성
점점 여러가지 접근방식에 대한 시야를 넓혀가게 되는것같다!
### 이번 주차 스터디하면서 어려웠던 점 작성
자바스크립트 내장함수 이용해서 간결하게 쓰는게 힘들다
### 오늘 회고 때 가장 인사이트가 있었던 점
기출문제 푸신분들이 생각보다 많아서 놀랬다..대단하신분들bb
### action plan: 회고를 통해 얻은 다음 주차 스터디 때 적용하고 싶은 행동 방안 작성
좀더 자세히 수도코드를 작성해볼까한다

**Reply by Sunmon** [reply]
- 이번 주차 스터디하면서 좋았던 점 작성: 
  - 숫자인걸 어떻게 판단할수 있는지 고민했었는데 정규식을 쓰는 방법이 있었네요.
  - 잊고 있던 템플릿 문자열을 리뷰하면서 기억해냈습니다 ㅋㅋ 
- 이번 주차 스터디하면서 어려웠던 점 작성:
  - 실전문제 풀이가 어려웠습니다. 한 문제 푸는데 두시간이 걸리니...
- 오늘 회고 때 가장 인사이트가 있었던 점:
  - 동빈북 챕터 12 문제가 릿코드 미디엄보다 어려운거같습니다 
- 스터디 건의사항
- action plan: 회고를 통해 얻은 다음 주차 스터디 때 적용하고 싶은 행동 방안 작성
  - 릿코드를 더 풀려고 합니다 
  - 모각알 여는 시간이 랜덤인가요 참여하고싶은데 ㅎㅎㅎ

**Reply by sooster910** [reply]
**이번 주차 스터디하면서 좋았던 점 작성**
좋앗던 점은 이 스터디 마저 없었다면 3개라도 문제라도 못풀었을 겁니다. 
** 이번 주차 스터디하면서 어려웠던 점 작성 **
이번주엔  저번주 보다 알고리즘 풀이에 여유가 좀 없었던게 사실입니다.  할일들을 골고루 잘 배분해서 하는게 쉽지 않습니다. ㅠ시간 분배가 제일 어렵네요.

**오늘 회고 때 가장 인사이트가 있었던 점** 
동동님의  정수 부분 설명으그리고 sunmon 님 풀이 보면서 다른 방식으로 접근하시는 부분도 좋았습니다. sumon님꺼는 아직 더 보고있습니다

**스터디 건의사항**
action plan: 회고를 통해 얻은 다음 주차 스터디 때 적용하고 싶은 행동 방안 작성
모각알에 참여하고 싶다면 참여해야겠습니다!

**Reply by hyunhaha** [reply]
### 좋았던 점
여러 가지 방법으로 푼 코드를 볼 수 있어 좋았습니다. (js로 작성된..!)

### 어려웠던 점
다른 할일 한다고 자꾸 미루다보니 목표치까지 다 못해서.. 정말 시간을 정해 놓고 풀어봐야 할 것 같아요

### 오늘 회고 때 가장 인사이트가 있었던 점
책에 나온 풀이 말고 새롭게 푸신 분들의 풀이를 보고 배울 수 있었습니다.

### action plan: 회고를 통해 얻은 다음 주차 스터디 때 적용하고 싶은 행동 방안 작성
하루에 한문제..? 이틀에 두문제 정도 정해서 풀어야겠습니다

**Reply by skt-t1-byungi** [reply]
리밋
- 좋은점: 요번주차 프로그래머스 문제는 예전에 풀어본게 많아서... 예전에 푼거
- 어려운점: 약간 텐션이 좀 떨어진것;  짐까지 맛보기인뎅 다음장부터 어려워지면 더 떨어질수도?!
- 액션플랜: 텐션올리기 ㅎㅎ;

**Reply by LeeGeonwoo22** [reply]
## 이번 주차 스터디하면서 좋았던 점 작성
부트캠프를 수료하고, 알고리즘에 대해  막연히 '공부해야지.. 언젠간 해야지' 라는 생각으로 차일 피일 미뤄왔었습니다. 하지만 스터디 그룹을 이번에 가입하게 되면서 ,  중간 스케줄을 하느라 피곤한 와중에도 
긴장의 끈을 놓지않고 공부도 하고 무엇보다 알고리즘 기초적인 부분에대해 찾아보기도 하는 그런 주였습니다.  
## 이번 주차에 아쉬웠던점은 ?
일단 문제를 모두 풀지못해 아쉽습니다. 정말 다음엔 한문제라도 좋으니 풀어보고싶네요.
## 오늘 회고때 가장인사이트가 있었던점?
아무래도 기초인것 같습니다. 알고리즘이란 것을 풀기위한 기본적인 것들.. 
유투브를 통해 어떻게 공부하고 접근할것인지에 대해 봤었는데 다른분들의 수도코드를 보고 
직접 내가 옮겨보고 왜 그렇게 작성하였는지 복습해보면 더 좋을것 같단 생각도 들었습니다. 
## action plan: 회고를 통해 얻은 다음 주차 스터디 때 적용하고 싶은 행동 방안 작성
따로 없습니다..ㅎㅎ 앞으로 열심히 배우겠습니다 감사합니다.

**Comment by bigsaigon333** [comment]
## 조별 회고 공유: ~23:00
회고 작성 조별 공유

소회의실1: HH, sue, Sunmon, 동동
소회의실2: 리밋, 연리사, 이건우 호준

**Comment by bigsaigon333** [comment]
## 체크아웃

조별 공유후 체크아웃 작성하시고 바로 해산하시면 되요~~

회고 종료후의 기분을 10점 만점으로 나타낸다면~? :smile:

**Reply by Sunmon** [reply]
7: 휴일에 마냥 논건 아니고 뭐라도 한 일이 있어서 다행입니다 ㅎㅎ

**Reply by skt-t1-byungi** [reply]
8: 낼은 할건하고 쉬어야지 ㅎㅎ

**Reply by bigsaigon333** [reply]
## 동동 8점

많은 분들이 모각알에 적극적으로 참여의사를 보여주셔서 너무 좋습니다!!
빨리 가려면 혼자 가고, 멀리 가려면 같이 가라고 했는데...
혼자서는 가지를 못하네요 ㅠ  같이 멀리 빨리 갑시다~~ :smile:

**Reply by Yeonlisa** [reply]
9점 : 배운게 많았습니다!!근데 코드리뷰 conversation이 많을수록 좋다고 생각되는건 제 기분이겠쪄..?ㅠ

**Reply by ganeodolu** [reply]
8: 주말이 후딱 지나갔네요 스터디가 의무감이 되지 않도록 노력이 필요한 것 같아요 텐션업!!ㅎㅎ

**Reply by LeeGeonwoo22** [reply]
10 : 첫 스터디주라 배운것도 많습니다.. ㅎㅎ 잘부탁드립니다.

**Reply by sooster910** [reply]
7 : 문제는 얼마 풀지 못해도 회고를 하니 왠지..회개하는 기분이 드네요

**Comment by bigsaigon333** [comment]
@ganeodolu  텐션업 텐션업!!!!!
